README.md

```diff
@@ -37,18 +37,94 @@ The latest version from the release branch 'master' is automatically deployed to
 
 $ `git clone https://github.com/ita-social-projects/ZeroWaste.git`
 
-## Setup
+## Local setup
+First of all you need rvm to setup project. For the operating system Windows the optimal solution is to use <a href="https://docs.microsoft.com/en-us/windows/wsl/">WSL</a>.
 
 $ `bin/setup`
-or 
-$ `bundle install` 
+or
+$ `bundle install`
+
+<b>PostgreSQL</b>
```

Comment:
In the project, the standard password for Postgresql is "password", maybe add information that you need to change the password in the project or set the postgresql password to "password"

![photo_2022-07-22_13-10-27](https://user-images.githubusercontent.com/93429979/180427359-fe276b57-e4f0-4be5-a9cc-d432e2a05dbe.jpg)